### DISCUSSION TITLE
파이썬 빈 배열 생성 차이 질문

### DISCUSSION THREAD

**Discussion by injoon2019** [woowacourse-study/2022-lv3-algorithm-study#120] [문제 같이 풀어요~]
``` python
teleports = [[] for _ in range(N+1)]
teleports = [[]] * (N+1)
```
위 코드에서 두 방식이 어떤 차이가 있을까요?
아래 방식으로 하니 출력 결과는 동일한데 아래 방식은 시간 초과가 나고 위의 방식은 통과하네요

![image](https://user-images.githubusercontent.com/46641538/179965607-485ab42b-65b3-493b-bf9b-022d4ab5d61b.png)

**Comment by sinb57** [comment]
배열의 요소를 N개 생성하기 위해, 질문 주신 두 방식을 많이들 사용하는데,
Asterisk(*) 연산자를 이용할 때는 주의해야합니다!!  

<img width="304" alt="스크린샷 2022-07-23 오후 5 43 31" src="https://user-images.githubusercontent.com/22176552/180597751-50d632ad-5b68-42d7-9680-a67b2142f209.png">

**Reply by injoon2019** [reply]
오.. 쓰면서 이걸 몰랐네요 감사합니다 이프!!
